include/envoy/router/router.h

```diff
@@ -764,6 +764,8 @@ class RouteTracing {
    * @return the overall sampling percentage
    */
   virtual const envoy::type::FractionalPercent& getOverallSampling() const PURE;
+
+  virtual const std::vector<Tracing::CustomTagPtr>& getCustomTags() const PURE;
```

Comment:
docs for all public interface functions here and below.

Reply:
commented